boot/aether/src/boot/aether.clj

```diff
@@ -68,7 +68,7 @@
 
 (defn transfer-listener
   [info]
-  (util/dbug "Aether: %s\n" (with-out-str (pprint/pprint info)))
+  (util/trace* "Aether: %s\n" (with-out-str (pprint/pprint info)))
```

Comment:
@arichiardi I've just tested with this branch and confirmed the `trace*` version is up to 100 times faster than the `dbug` version 👍